modules/syslog-configuring-acs.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-using-syslog-protocol.adoc
+:_module-type: PROCEDURE
+[id="syslog-configuring-acs_{context}"]
+= Configuring Syslog integration with {product-title}
```

Comment:
I don't think Syslog needs to be capitalized (here and sentence below), but I'm not going to hold up the PR over it.